src/biometryType.android.js

```diff
@@ -0,0 +1,12 @@
+import { NativeModules } from 'react-native';
```

Comment:
We don't need two different versions of the same file if we make both the `api's` promise compliant.